src/renderer/views/Subscriptions/Subscriptions.js

```diff
@@ -50,6 +61,11 @@ export default defineComponent({
         tabs.push('live')
       }
 
+      // community does not support rss
+      if (!this.hideSubscriptionsCommunity && this.activeSubscriptionList.length < 125) {
```

Comment:
RSS can also be manually enabled in the subscriptions settings

Reply:
True but when RSS fails it defaults to not using RSS for the other subscription lists

Reply:
But this isn't about whether RSS fails or not, if we can't get community posts from RSS then shouldn't we hide the tab when the user has selected RSS as the preferred method of getting the subscriptions?